packages/governance/src/governor/extensions/governor_votes_quorum_fraction.cairo

```diff
@@ -54,6 +54,8 @@ pub mod GovernorVotesQuorumFractionComponent {
         +Drop<TContractState>
     > of GovernorComponent::GovernorQuorumTrait<TContractState> {
         /// See `GovernorComponent::GovernorQuorumTrait::quorum`.
+        ///
+        /// Returns a percentage of the votes token total supply at a given timepoint in the past.
```

Comment:
Sounds a bit difficult to understand, maybe it could be rephrase

Reply:
Agree, tried to rephrase it a bit.